src/website/views/templates/order-checkout-flow.pug

```diff
@@ -0,0 +1,1697 @@
+      .o-header__start
+        span.o-header__title.-text--muted Business Everywhere
+      .o-header__end
+        .m-dropdown.-mr--1
+          button.a-btn.-flat.-inverse.-px--1.m-dropdown__trigger(data-position='bottom-end', id='support') Support
+          .m-dropdown__menu(style='width:10rem;')
+            a.m-dropdown__menu-item.-active(href='#') link 1
+            a.m-dropdown__menu-item(href='#') link 2
+            a.m-dropdown__menu-item(href='#') link 3
+        div
+          .a-avatar.-sm3.-grey
+            i(class='a-icon icon-user')
+        .m-dropdown.-mr--1
+          button.a-btn.-flat.-inverse.-px--1.m-dropdown__trigger(data-position='bottom-end', id='user') Tim Anderson
+          .m-dropdown__menu(style='width:10rem;')
+            a.m-dropdown__menu-item.-active(href='#') link 1
+            a.m-dropdown__menu-item(href='#') link 2
+            a.m-dropdown__menu-item(href='#') link 3
+  .a-divider.-inverse.-bg--black.-m--0
+  .-bg--black
+    ul.a-steps.-horizontalLabels.-inverse.-py--2.-m--0.-pr--4.-mw--640.-m--auto
+      li.-completed
+        div.a-steps__icon
+        a(href='#') Customer information
+        div.a-steps__line
+      li.-active
+        div.a-steps__icon
+        a(href='#') Configure locations
+        div.a-steps__line
+      li
+        div.a-steps__icon
+        a(href='#') Checkout
+  div.-d--flex
+    div(style='min-width:200px').-br--1.-d--flex.-flex--column.-justify-content--between
+      div
+        div.-d--flex.-flex--row.-justify-content--start.-pl--3.-py--1
+          h5 Locations
+          span#locations.-d--none.-my--1.-ml--1 (1)
+        ul.a-tabs.-vertical.-large
+          li#locationActiveTab.-active.-bg--grey-20
+              a(href='#') New Location
+          li#locationTab(style='display:none')
+            a(href='#').-text--larger
+              i.a-icon.icon-map-marker.-mr--1
+              | CenturyLink
+            div.-pl--6
+              p.-text--small.-mt--0
+                | 1100 112th Avenue <br />
+                | NE, Suite 400<br />
+                | Bellevue, WA 98004
+      div.-p--1.-d--flex.-flex--row.-justify-content--start.-align-items--center.-bt--1
+        i.a-icon.icon-circle-plus-outline.-sm3.-text--brand.-ml--1
+        h6.-text--brand.-ml--1 Add Location
+    .-px--5.-py--2.-m--auto.-flex--grow1.-mw--640(style='height:800px;overflow:auto;')#checkout
```

Comment:
This checkout div needs to be contained in a new parent div. You'll need to move the inline styles to the parent div and add a -flex--grow1 class. This will provide a less buggy experience and render a scroll on the parent instead of the panels.

Reply:
Something along these lines. But height: 800px; isn't scalable. Need to replace with 100% values instead of fixed heights.
![image](https://user-images.githubusercontent.com/130654/53536919-00e8d680-3abd-11e9-8e3b-35e324072912.png)

Reply:
done